build.sbt

```diff
@@ -1,6 +1,6 @@
 name := """capillary"""
 
-version := "1.2"
+version := "2.0.0"
```

Comment:
FYI - we need to update internal deployment and `stats-to-datadog.py` automation for this version change. It's currently hardcoded to `1.2`.